samples/java-application-with-reflection/pom.xml

```diff
@@ -138,27 +138,40 @@
                         </executions>
                         <!-- tag::native-plugin-agent-options[] -->
                         <configuration>
-                            <!-- end::native-plugin-agent-options[] -->
```

Comment:
You have removed Asciidorctor tags that are used to render snippets in the documentation